lampadaires/data_lampadaires/lampadaires.qml

```diff
@@ -0,0 +1,278 @@
+<!DOCTYPE qgis PUBLIC 'http://mrcc.com/qgis.dtd' 'SYSTEM'>
+<qgis version="2.14.3-Essen" minimumScale="0" maximumScale="1e+08" simplifyDrawingHints="0" minLabelScale="0" maxLabelScale="1e+08" simplifyDrawingTol="1" simplifyMaxScale="1" hasScaleBasedVisibilityFlag="0" simplifyLocal="0" scaleBasedLabelVisibilityFlag="0">
```

Comment:
Very side node, but can you export the QML from the project to this file ?
QGIS 2.14 is very old, so just to update it from time to time.
Or I think you should drop it, the QML is already in the project anyway.